src/test/java/uk/gov/companieshouse/api/testdata/service/impl/TestDataServiceImplTest.java

```diff
@@ -893,6 +892,7 @@ void createAcspMembersData() throws DataException {
         verifyAcspMembersData(result, "memberId", "acspNumber", "userId", "active", "role");
         verify(acspProfileService).create(any(AcspProfileSpec.class));
         verify(acspMembersService).create(any(AcspMembersSpec.class));
+        verify(acspProfileService).create(argThat(profile -> profile.getAmlDetails() == null));
```

Comment:
If we are adding this verify statement we could remove the one two lines before, in 893